### PR TITLE
Clean up warnings

### PR DESCRIPTION
**Pull request by cha0sg0d**
This PR cleans up two warnings:

## Warning 1
CleanShot 2023-05-09 at 10 40 05@2x

Fixed by recommendation [here](https://github.com/vercel/next.js/issues/40687#issuecomment-1421526821)

## Warning 2
```
warn  - ../../node_modules/.pnpm/web-worker@1.2.0/node_modules/web-worker/cjs/node.js
Critical dependency: the request of a dependency is an expression

Import trace for requested module:
../../node_modules/.pnpm/web-worker@1.2.0/node_modules/web-worker/cjs/node.js
../../node_modules/.pnpm/ffjavascript@0.2.56/node_modules/ffjavascript/build/main.cjs
../../node_modules/.pnpm/snarkjs@0.5.0/node_modules/snarkjs/build/main.cjs
../nymjs/build/utils.js
../nymjs/build/lib.js
```

Fixed by recommendation [here](https://typeorm.io/faq#how-to-use-webpack-for-the-backend)

### CODE REVIEW

File: packages/frontend/next.config.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 const nextConfig = {
   reactStrictMode: true,
   transpilePackages: ['react-syntax-highlighter', 'swagger-client', 'swagger-ui-react'],
   webpack: (config) => {
     config.resolve.fallback = { fs: false };
     config.experiments = { asyncWebAssembly: true, layers: true };
-
+    config.plugins.push(
+      new FilterWarningsPlugin({
+        exclude: [/Critical dependency/],
```
Comment (lsankar4033) [inline packages/frontend/next.config.js:12]: it seems like the specific warning we want to suppress is around dependencies that are expressions. Is it possible to scope the exclude statement to that?

I can't reproduce the warning, but maybe something like `/Critical dependency  the request of a dependency is an expression/` is sufficient to suppress the warning that's annoying us

Wary of over-suppressing.

**Review by lsankar4033 — CHANGES_REQUESTED** [review]
I actually wasn't able to reproduce either of these warnings on main. In either vscode or the running site.

But your fixes make sense logically! If they fix for you, I'd merge in. Just the one note about over-suppressing the Critical dependency warning

**Review by lsankar4033 — CHANGES_REQUESTED** [review]
I actually wasn't able to reproduce either of these warnings on main. In either vscode or the running site.

But your fixes make sense logically! If they fix for you, I'd merge in. Just the one note about over-suppressing the Critical dependency warning

File: packages/frontend/next.config.js (hunk 1 above)
Comment (cha0sg0d) [reply packages/frontend/next.config.js:12]: Good point, I'll test that out. Don't want to oversurpress.

Comment (cha0sg0d) [reply packages/frontend/next.config.js]: Fixed by [1492c97](https://github.com/personaelabs/nym/pull/67/commits/1492c97a1504ab36a95071f0cec3dbe1af02e448)

**Review by lsankar4033 — APPROVED** [review]

**Pull request merged**